### PR TITLE
Rephrase "Uploading without Debug IDs" a bit

### PR DESCRIPTION
**Pull request by lforst**
- Flips the requirements
- Explains what we mean by debug IDs
- Move some sentences around
- Fix broken links

### CODE REVIEW

**Review by shanamatthews — COMMENTED** [review]
I'm confused about when to use the new guide vs. old.

This states that if you're using an older bundler plugin, you should use this guide, but also states that the bundler plugins are applicable to any version.

So if I were using v 2.1.0 of `@sentry/webpack-plugin`, does the normal webpack guide work for me?

File: src/platform-includes/sourcemaps/uploading-without-debug-ids/javascript.mdx [hunk 1]
```diff
@@ -56,18 +60,8 @@ sentry-cli sourcemaps upload --release=<release_name> /path/to/directory
...
-### Finalize Release
-
-At this point, the release is in a draft state (“_unreleased_”). Once all artifacts have been uploaded and your app has been published successfully, finalize the release with the following command:
-
-```bash
-sentry-cli releases finalize <release_name>
-```
-
-For more information on how to configure releases via Sentry CLI, see our [Sentry CLI release management docs](/product/cli/releases/).
+Open up Sentry and navigate to **[Settings] » Projects » Select your project » Source Maps** on Sentry.
```
Comment (shanamatthews) [inline src/platform-includes/sourcemaps/uploading-without-debug-ids/javascript.mdx:63]:
```suggestion
Open up Sentry and navigate to **[Settings] > Projects > Select your project > Source Maps** on Sentry.
```

File: src/platform-includes/sourcemaps/uploading-without-debug-ids/javascript.mdx [hunk 2]
```diff
@@ -56,18 +60,8 @@ sentry-cli sourcemaps upload --release=<release_name> /path/to/directory
...
 
 ## Summary
 
-It can come with some headaches, but you can still upload source maps regardless of what version of our SDK or bundler plugin (or sentry-cli) your application depends on. Visit [Troubleshooting Source Maps](/platforms/javascript/sourcemaps/troubleshooting_js/) if you experience any problems following our guides.
+It can come with some headaches, but you can still upload source maps regardless of what version of our SDK or bundler plugin (or sentry-cli) your application depends on. Visit <PlatformLink to="/sourcemaps/troubleshooting_js">the regular Source Maps guide</PlatformLink> if you experience any problems following our guides.
```
Comment (shanamatthews) [inline src/platform-includes/sourcemaps/uploading-without-debug-ids/javascript.mdx:67]:
```suggestion
You can still upload source maps regardless of what version of our SDK or bundler plugin (or sentry-cli) your application depends on. Visit the regular <PlatformLink to="/sourcemaps/troubleshooting_js">Source Maps</PlatformLink> guide if you experience any problems following this guide.
```

File: src/platform-includes/sourcemaps/uploading-without-debug-ids/javascript.mdx [hunk 3]
```diff
@@ -1,26 +1,30 @@
...
+If your application uses an old version of our SDK or bundler plugin (or sentry-cli), the result will roughly be the same. You can still upload source maps; however, the process will differ. Instead of using Debug IDs, you'll be using releases, to match an events stack frame with its corresponding minified source and source map file (known as artifacts).
 
 ## Uploading Source maps using our Bundler Plugin
 
-Our guides on uploading source maps with our bundler plugins are applicable to any version your application might use. Visit [Uploading Source Maps](/platforms/javascript/guides/react/sourcemaps/uploading/) to find an applicable guide.
+Our guides on uploading source maps with our bundler plugins are applicable to any version your application might use. Visit <PlatformLink to="/sourcemaps">the regular Source Maps guide</PlatformLink> to find an applicable guide.
```
[the remark this answers is not in the document]
Comment (lforst) [reply src/platform-includes/sourcemaps/uploading-without-debug-ids/javascript.mdx:27]: dang that's not even true anymore - whoops. Thanks for pointing that out!

**Review by Lms24 — APPROVED** [review]

**Comment by lforst** [comment]
> I'm confused about when to use the new guide vs. old.
> 
> This states that if you're using an older bundler plugin, you should use this guide, but also states that the bundler plugins are applicable to any version.
> 
> So if I were using v 2.1.0 of @sentry/webpack-plugin, does the normal webpack guide work for me?

I more or less rewrote this page trying to make this more clear:

- Users should read this guide if any of the bullet points apply to them
- Then they just pick their tooling. It's either
	- CLI
	- Webpack v1.x
	- Bundler plugins v2.x (bundler plugins pre 2.x don't exist)

File: src/platform-includes/sourcemaps/uploading-without-debug-ids/javascript.mdx [hunk 4]
```diff
@@ -1,28 +1,21 @@
...
+If you're using one or more of these older versions of the tools, you can still upload source maps; however, the process will differ. Instead of using debug IDs, you'll use releases to match an event's stack frame with its corresponding minified source and source map file (known as artifacts).
 
-## Uploading Source maps using our Bundler Plugin
-
-Our guides on uploading source maps with our bundler plugins are applicable to any version your application might use. Visit [Uploading Source Maps](/platforms/javascript/guides/react/sourcemaps/uploading/) to find an applicable guide.
-
-## Uploading Source Maps using `sentry-cli`
+## Uploading using `sentry-cli`
 
 Here is our guide to uploading artifacts using [releases](/product/releases/)
```
Comment (shanamatthews) [inline src/platform-includes/sourcemaps/uploading-without-debug-ids/javascript.mdx:20]:
```suggestion
```

**Comment by lforst** [comment]
> I think I'd been staring at this page too long.

@shanamatthews  Same... For some reason, it is super hard for me to wrap my head around what users would like to get out of docs for this topic. Thank you so much for taking a look!

**Pull request merged**